scripts/analyze.sh

```diff
@@ -92,6 +92,42 @@ fi
 check_for_js_require
 
 # Check for native modules
+if check_for_require net || check_config_file ZJS_NET; then
+    >&2 echo Using module: net
+    MODULES+=" -DBUILD_MODULE_NET -DBUILD_MODULE_BUFFER -DBUILD_MODULE_EVENTS"
+    echo "export ZJS_NET=y" >> zjs.conf.tmp
```

Comment:
the files have bee placed with $PRJFILE, $ARCPRJFILE, and $CONFFILE, please use these variables